src/app/wallet/shared/transaction.service.ts

```diff
@@ -60,24 +60,26 @@ export class TransactionService {
   */
 
   rpc_update() {
-    this.rpc.call('listtransactions', [
-      '*', +this.MAX_TXS_PER_PAGE,
-      ((this.currentPage ? this.currentPage - 1 : 0) * this.MAX_TXS_PER_PAGE)
-    ])
-    .subscribe(
-      (txResponse: Array<Object>) => {
-        // The callback will send over an array of JSON transaction objects.
-        this.log.d(`rpc_loadTransactions_success, supposedly tx per page: ${this.MAX_TXS_PER_PAGE}`);
-        this.log.d(`rpc_loadTransactions_success, real tx per page: ${txResponse.length}`);
-
-        if (txResponse.length !== this.MAX_TXS_PER_PAGE) {
-          this.log.er(`rpc_loadTransactions_success, TRANSACTION COUNTS DO NOT MATCH (maybe last page?)`);
-        }
-
-        txResponse.forEach((tx) => {
-          this.addTransaction(tx);
+    setTimeout(() => {
```

Comment:
This is a hack, we should just fix the timing rather than add an unnecessary timeout